spec/support/shared_examples/busca_logotipo.rb

```diff
@@ -0,0 +1,14 @@
+shared_examples_for 'busca_logotipo' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.